addons/sys_radio/initSettings.sqf

```diff
@@ -0,0 +1,21 @@
+[{!isNil EFUNC(api,getAllRadios)},{
```

Comment:
Why would that ever not be available?

Ideally that function should be in `sys_core` and `api` should just be a wrapper with extra arguments checks. But this is not the only bad example of this in the current code base.